pkg/security/podsecuritypolicy/seccomp/strategy_test.go

```diff
@@ -293,6 +350,14 @@ func TestValidateContainer(t *testing.T) {
 			},
 			expectedError: "Forbidden: bar is not an allowed seccomp profile. Valid values are foo",
 		},
+		"valid container field and no annotation, required profiles": {
+			pspAnnotations: allowSpecificLocalhost,
+			seccompProfile: &api.SeccompProfile{
+				Type:             api.SeccompProfileTypeLocalhost,
+				LocalhostProfile: &foo,
+			},
+			expectedError: "",
+		},
 	}
```

Comment:
add a negative test that makes sure a container localhost profile field of bar would be forbidden

Reply:
Added as suggested :+1: